thehive/app/org/thp/thehive/models/Tag.scala

```diff
@@ -54,3 +56,16 @@ object Tag {
     }
   }
 }
+
+case class RichTag(tag: Tag with Entity) {
```

Comment:
Creating a RichTag class is useless if there is no extra field, compared with Tag with Entity,